components/kyma-metrics-collector/pkg/keb/client.go

```diff
@@ -0,0 +1,133 @@
+package keb
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	kebruntime "github.com/kyma-project/control-plane/components/kyma-environment-broker/common/runtime"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/util/retry"
+)
+
+type Client struct {
```

Comment:
you can use: https://github.com/kyma-project/control-plane/blob/master/components/kyma-environment-broker/common/runtime/client.go

Reply:
Thanks @piotrmiskiewicz, since we need to roll-out the new component ASAP, we gonna take care of this in a follow-up issue.